### PR TITLE
files added

### PR DESCRIPTION
**Pull request by Shavvimal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Installation:
+  Clone or download the repo.
+  cd into the server directory.
+  Run npm install to install the dependencies.
+  (If this does not work, you may have to install the following dependencies: express, cors, and nodemon. Change the package.json main:"index.js" to "app.js" 
+  and add the following to the script section :
```
Comment (Shavvimal) [inline README.md:6]: when cloning the repo, the package.json has these scripts

File: README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+Installation:
+  Clone or download the repo.
+  cd into the server directory.
+  Run npm install to install the dependencies.
```
Comment (Shavvimal) [inline README.md:4]: `bar ticks` make the md display it like code

File: README.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    "dev": "nodemon app.js"
+  },)
+  Move onto usage.
+  
+Usage
+  In the server directory, run nodemon app.js to start the server.
+  Navigate to the client directory and open the index.html using http server or live server extension
+  Two buttons have been created, they do seperate things, have a go!
+
+Future features
```
Comment (moony98) [inline README.md:18]: like that you implemented the future features

File: index.html [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    <title>Google search engine replica</title>
+    <link rel="stylesheet" href="style.css" />
+    <script defer src="main.js"></script>
+  </head>
+  <body>
+    <main>
+      <section>
+        <img src="images/google.png" alt="" class="google" />
+        <div class="searchBox">
+          <input type="text" name="" class="query" />
```
Comment (moony98) [inline index.html:16]: empty fields?

File: index.html [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+          <div class="icons">
+            <div><img src="images/search.png" alt="" /></div>
+            <div><img src="images/mic.png" alt="" /></div>
+          </div>
+        </div>
+        <div class="button">
+          <button class="searchBtn">Google Search</button>
+          <button>
+            <a href="https://www.youtube.com/watch?v=dQw4w9WgXcQ"
```
Comment (Shavvimal) [inline index.html:25]: you got me :(

File: main.js [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+let newsearch;
+const list = document.querySelector(".myheadhurts");
+let main = document.querySelector("main");
+let section = document.querySelector("section");
+let img = document.querySelector(".google");
+let searchBox = document.querySelector(".searchBox");
+let searchBtn = document.querySelector(".searchBtn");
+
+async function results() {
```
Comment (Shavvimal) [inline main.js:12]: nice

File: main.js [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+  let search = await fetch("http://localhost:3000");
+  newsearch = await search.json();
+  addContent(newsearch);
+}
+
+function addContent(newsearch) {
```
Comment (moony98) [inline main.js:18]: Great method of pulling all the data

File: server/app.js [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+    header: "MY TV",
+    text: "Place a piece of black tape over the remote sensor of the TV - No one would be able to change channels with the remote!",
+  },
+];
+
+const resultURL = "https://www.fun-stuff-to-do.com/good-pranks.html";
+
+app.get("/", (req, res) => res.send(results));
+
+app.listen(port, () =>
```
Comment (Shavvimal) [inline server/app.js:55]: would be better to tell the server to listen out for requests in a separate file, index.js for example (idk why either, just "convention" )

File: server/app.js [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+  console.log(`Express departing now from http://localhost:${port}!`)
```
Comment (Shavvimal) [inline server/app.js:56]: remove the ! or add a space, as it makes the link in VS code useless

File: server/app.js [hunk 10]
```diff
@@ -0,0 +1,57 @@
+const express = require("express");
+const app = express();
+const cors = require("cors");
+const port = 3000;
+const path = require("path");
+app.use(cors());
+
+const results = [
```
Comment (Shavvimal) [inline server/app.js:8]: would be cleaner to host your data in a separate data.js file for example. Remember to export this though so we can have access to the data from the other parts of the app

File: server/package.json [hunk 11]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "server",
+  "version": "1.0.0",
+  "main": "app.js",
+  "scripts": {
+    "start": "node app.js",
```
Comment (Shavvimal) [inline server/package.json:6]: 👍

File: README.md (hunk 2 above)
Comment (Shavvimal) [reply README.md:4]: `this is code`

**Pull request merged**